core/sodasql/cli/cli.py

```diff
@@ -153,6 +171,8 @@ def create(warehouse_type: str,
         logger.info("If you think this is a bug in Soda SQL, please open an issue at: "
                     "https://github.com/sodadata/soda-sql/issues/new/choose")
         sys.exit(1)
+    finally:
+        trace.get_current_span().end()
```

Comment:
I don't think we need this anymore .. right ? traces should be set to auto-end when we exit ...